knowledge-portal/source/README.md

```diff
@@ -0,0 +1,36 @@
+# Usage
+
+## Adding this pipeline to your repo
+
+You will have to modify the existing `gitlab-ci.yml` file to include:
+
+```yaml
+    stages:
+        - copy_to central_repo
+    include:
+        - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+
+```
+## Configure CI/CD variables
+
```

Comment:
You can add a note here that the steps of this section can ideally be performed on the overall gitlab instance, so that it need not be replicated for every repo/